SR2021/2021-01-01-january-update.md

```diff
@@ -0,0 +1,48 @@
+---
+to: SR2021 teams
+subject: Student Robotics January Update
+---
+
+Hello there!
+
+There is less than 2 weeks until our [first round of league matches](https://studentrobotics.org/events/sr2021/league-1/) so we wanted to check in and give you an update.
+
+## League Sessions
+
+The first league session will be happening on the 16th of January, at 12:00 UK time, lasting around 1.5 hours. You will need to have submitted your code to us (more on that below) by 09:00 UK time on the same day. If you do not submit code, or submit late, you will be marked absent and score 0 points.
+
+Further details, and a link to the livestream, will be on our website soon - for this and the other league sessions.
+
+## Friendlies
+
+As noted in our previous email, we're going to be running some friendly matches on the 9th of January. These matches are entirely optional, and will not contribute to the league. The matches are simply a chance to see how your robot performs against other robots, as well as socialise with other teams and us volunteers. Unlike our other events, these matches will be incredibly relaxed and informal. We encourage anyone to join in the conversation in Discord or in the chat.
+
+[Watch our livestream](https://youtu.be/iwa-7R6sR8M).
```

Comment:
Do we really want to livestream the friendlies? Not sure if that's what this line means, however it seems counter to "relaxed and informal" to then also be livestreaming them.

Reply:
It's an unlisted video. YouTube is the easiest way to get the footage out and have a good feedback loop between teams watching live.

Reply:
Ah, I think possibly I'm misunderstanding the goal here. I had assumed that friendlies were much more akin to Tinker Time -- that we'd probably be giving the teams their logs and the recordings without any fanfare, to allow them to debug stuff having interacted with another robot. Is something else the goal here?

Perhaps it would be useful to clarify what form the livestream will take -- are we expecting to have commentary on it for example? If it's just that we're going to use the livestream to distribute the videos I think we should say that explicitly.

Reply:
There will be a commentary from any volunteer who's around, much like mentoring at a tech day. There won't be any play-by-play commentary on a match, or at least far from the same format of a regular competition.

My expectation is basically for it to be a livestreamed Meet call with some volunteers, where we show the matches, chat with competitors, comment on performance, ask questions, things like that.